recipe/meta.yaml

```
@@ -19,28 +17,24 @@ build:
     - splk-py-trace-bootstrap = splunk_otel.cmd.bootstrap:run_deprecated
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
```
Extra verbosity isn't needed in most cases